ladybug_comfort/collection/adaptive.py

```diff
@@ -0,0 +1,431 @@
+
+    @property
+    def degrees_from_neutral(self):
+        """Data Collection of the degrees from desired neutral temperature in C."""
+        return self._build_coll(self._degrees_from_neutral,
+                                OperativeTemperatureDelta(), 'C')
+
+    @property
+    def is_comfortable(self):
+        """Data Collection of integers noting whether the input conditions are
+        acceptable according to the assigned comfort_parameter.
+
+        Values are one of the following:
+            0 = uncomfortable
+            1 = comfortable
+        """
+        return self._build_coll(self._is_comfortable, ThermalComfort(), 'condition')
+
+    @property
+    def thermal_condition(self):
+        """Data Collection of integers noting the thermal status of a subject
+        according to the assigned comfort_parameter.
+
+        Values are one of the following:
+            -1 = cold
+             0 = netural
+            +1 = hot
+        """
+        return self._build_coll(self._thermal_condition, ThermalCondition(), 'condition')
+
+    @property
+    def cooling_effect(self):
+        """Data Collection of the cooling effect of the air speed in degrees Celcius.
+
+        This is the difference between the air temperature and the
+        adjusted air temperature [C].
+        """
+        return self._build_coll(self._cooling_effect, OperativeTemperatureDelta(), 'C')
+
+    @property
+    def percent_comfortable(self):
+        """The percent of time comfortabe given by the assigned comfort_parameter."""
+        return (sum(self._is_comfortable) / self._calc_length) * 100
+
+    @property
+    def percent_uncomfortable(self):
+        """The percent of time uncomfortabe given by the assigned comfort_parameter."""
+        return 100 - self.percent_comfortable
+
+    @property
+    def percent_neutral(self):
+        """The percent of time that the thermal_condiiton is neutral."""
+        _vals = [1 for x in self._thermal_condition if x == 0]
+        return (sum(_vals) / self._calc_length) * 100
+
+    @property
+    def percent_cold(self):
+        """The percent of time that the thermal_condiiton is cold."""
+        _vals = [1 for x in self._thermal_condition if x == -1]
+        return (sum(_vals) / self._calc_length) * 100
+
+    @property
+    def percent_hot(self):
+        """The percent of time that the thermal_condiiton is hot."""
+        _vals = [1 for x in self._thermal_condition if x == 1]
+        return (sum(_vals) / self._calc_length) * 100
+
+
+class PrevailingTemperature(object):
+    """Determine prevailing temperature from annual DataCollections of outdoor temperature.
+
+    Properties:
+        outdoor_temperature
+        avg_month_or_running_mean
+        hourly_prevailing_temperature
+        daily_prevailing_temperature
+        monthly_prevailing_temperature
+        monthly_per_hour_prevailing_temperature
+    """
+
+    def __init__(self, outdoor_temperature, avg_month_or_running_mean=True):
```

Comment:
I had originally done this because I wanted be consistent with the adaptive comfort parameter input, which needs to be explicit because there are a number of other options that are structured as `this_or_that`.  I guess when it shows up alone like this, I can just do what you suggest.